support-payment-api/src/main/scala/backend/BackendError.scala

```diff
@@ -39,6 +42,7 @@ object BackendError {
   final case class AmazonPayApiError(err: AmazonPayError) extends BackendError
   final case class SubscribeWithGoogleDuplicateEventError(error: SubscribeWithGoogleDuplicateInsertEventError) extends BackendError
   final case class Email(error: EmailService.Error) extends BackendError
+  final case class TrackingError(err: Throwable) extends BackendError
   final case class MultipleErrors(errors: List[BackendError]) extends BackendError
```

Comment:
`MultipleErrors` is now only being used by SubscribeWithGoogle, which needs to be removed. But we'll do this in a follow up PR